setup.py

```diff
@@ -21,6 +21,7 @@
     install_requires=[
         'basic_modeling_interface',
         'esmvaltool',
+        'grpc4bmi>=0.2.12,<0.3',
```

Comment:
I think this is already part of master, no?

Reply:
Good catch, yes, it is sorted now.